dashboard/src/app/factories/create-factory/action/factory-action-box.html

```diff
@@ -14,43 +14,46 @@
                    che-place-holder="Enter value"
                    aria-label="Enter value"
                    required
+                   ng-maxlength="255"
                    ng-model="factoryActionBoxCtrl.selectedParam">
           <div ng-message="required">Param is required.</div>
+          <div ng-message="maxlength">The value has to be less than 255 characters long.</div>
```

Comment:
> doest it require as well md-maxlength ?

AFAIK, directive `ngMaxlength` adds `maxlength` validator to `ngModel`,
so, it doesn't require any other validator

> 255 or 256

I've tried to add action with length equal 256 symbols and got the  error.